examples/simple-trino-cluster-hive-ha-s3.yaml

```diff
@@ -0,0 +1,84 @@
+# stackablectl operator install commons secret hive zookeeper
+# helm install minio minio --repo https://charts.bitnami.com/bitnami --set auth.rootUser=minio-access-key --set auth.rootPassword=minio-secret-key
+---
+apiVersion: trino.stackable.tech/v1alpha1
+kind: TrinoCluster
+metadata:
+  name: simple-trino
+spec:
+  version: 396-stackable0.2.0
+  catalogLabelSelector:
+    matchLabels:
+      trino: simple-trino
+  coordinators:
+    roleGroups:
+      default:
+        replicas: 1
+  workers:
+    roleGroups:
+      default:
+        replicas: 1
+---
+apiVersion: trino.stackable.tech/v1alpha1
+kind: TrinoCatalog
+metadata:
+  name: hive
+  labels:
+    trino: simple-trino
+spec:
+  connector:
+    hive:
+      metastore:
+        configMap: simple-hive-derby
+      s3:
+        reference: minio
+---
+apiVersion: hive.stackable.tech/v1alpha1
+kind: HiveCluster
+metadata:
+  name: simple-hive-derby
+spec:
+  version: 3.1.3-stackable0.2.0
+  metastore:
+    roleGroups:
+      default:
+        replicas: 2
+        config:
+          database:
+            connString: jdbc:derby:;databaseName=/tmp/metastore_db;create=true
```

Comment:
Please don't :D
Hive HA does only work if they share a Postgres below. If we use a derby here, bad things are going to happen. There should be other examples using a Postgres here